### ISSUE TITLE
Ensure fundamental accessibility for basic login flow

### ISSUE THREAD

**Issue by mabartos** [keycloak/keycloak#11313]
### Describe the bug

When the login flow consists of UsernameForm and PasswordForm, the user with a screen reader cannot restart the login flow, because everything that's mentioned on the PasswordForm page is "Password, Sign in, Link".

It should be "Password, Sign in, Restart login".

### Version

999

### Expected behavior

Screen reader says: "Password, Sign in, Restart login"

### Actual behavior

Screen reader says: "Password, Sign in, Link"
